modules/mono/editor/bindings_generator.cpp

```diff
@@ -2536,13 +2536,7 @@ void BindingsGenerator::_default_argument_from_variant(const Variant &p_val, Arg
 
 	switch (p_val.get_type()) {
 		case Variant::NIL:
-			if (ClassDB::class_exists(r_iarg.type.cname)) {
-				// Object type
-				r_iarg.default_argument = "null";
-			} else {
-				// Variant
-				r_iarg.default_argument = "null";
-			}
+			r_iarg.default_argument = "null";
```

Comment:
Can we have this comment: `// Either Object type or Variant`

Reply:
Sure.